cmd/config/validate.go

```diff
@@ -59,13 +59,18 @@ func NewValidateCmd() *cobra.Command {
 
 			errs := cfg.Validate()
 			if len(errs) > 0 {
-				return fmt.Errorf("config validation failed: %w", errors.Join(errs...))
+				stderr := cmd.ErrOrStderr()
+				fmt.Fprintln(stderr, "Validation failed:")
+				for _, e := range errs {
+					fmt.Fprintln(stderr, "-", e.Error())
+				}
+				cmd.SilenceErrors = true
+				return errors.Join(errs...)
 			}
 
 			return nil
 		},
-		SilenceUsage:  true,
-		SilenceErrors: true,
```

Comment:
Wouldn't removing this line have been enough?

Reply:
Mostly yes, for testability the os.Stdin change also. The error formatting seems to be unnecessary since errors.Join already adds newlines.